docker/README.md

```diff
@@ -1,18 +1,28 @@
 # Get docker images
 
-You can either build the images locally, or get our "latest stable" version from the Docker hub. We recommend using the Docker hub option.
+You can either build the images locally, or get the images from the Docker hub. We recommend using the Docker hub option.
 
 ## Download images from the Docker hub
 
+**openSUSE images**
+
+    docker pull slindomansilla/openqa_data
+    docker pull slindomansilla/openqa_webui
+    docker pull slindomansilla/openqa_worker_default
```

Comment:
can we find an "opensuse" project on the docker hub? There might be one already. If not, at least use "suse"?